dev/CommonStyles/Button_themeresources.xaml

```diff
@@ -161,6 +161,9 @@
                                 <VisualState x:Name="Normal">
                                     <Storyboard>
                                         <PointerUpThemeAnimation Storyboard.TargetName="ContentPresenter" />
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ContentPresenter" Storyboard.TargetProperty="Foreground">
```

Comment:
>Foreground [](start = 139, length = 10)

Why not just add it to the ContentPresenter above similar to Background ? 

 <ContentPresenter
                        x:Name="ContentPresenter"
                        Background="{TemplateBinding Background}"
                        Foreground="{TemplateBinding Foreground}"

Reply:
That also works, I have updated the code. Thanks!